tests/Nox.Types.Tests/EntityFrameworkTests/Models/Country.cs

```diff
@@ -169,4 +169,9 @@ public sealed class Country
     /// Gets or sets the Flag.
     /// </summary>
     public Image Flag { get; set; } = null!;
+    
+    /// <summary>
+    /// Gets or sets the CitiesCounties.
+    /// </summary>
+    public Yaml CitiesCounties { get; set; } = null!;
```

Comment:
Can we rename the property to CitiesCountiesYaml?

Reply:
I would not prefer type as suffix in properties.